### PR TITLE
solution

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/components/Loader/PersonItem.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        <>
+          {(mother && typeof mother === 'object')
+            && <PersonLink person={mother} />}
+          {(mother && typeof mother === 'string') && `${mother}`}
+          {!mother && '-'}
```
Comment (polosanya) [inline src/components/Loader/PersonItem.tsx:40]: Can't you move this logic to findRelative function and just return a proper value from it to use it here?

File: src/components/Loader/PersonItem.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        </>
+      </td>
+      <td>
+        <>
+          {(father && typeof father === 'object')
+            && <PersonLink person={father} />}
+          {(father && typeof father === 'string') && `${father}`}
+          {!father && '-'}
```
Comment (polosanya) [inline src/components/Loader/PersonItem.tsx:48]: Same here

File: src/types/Error.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export type Error = {
+  message: string,
+  type: string;
+  isDanger?: boolean,
+};
```
Comment (polosanya) [inline src/types/Error.ts:5]: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
image

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost perfect!

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Pull request closed without merging**